msteams-platform/apps-in-teams-meetings/meeting-apps-apis.md

```diff
@@ -37,6 +37,550 @@ The following table provides a list of APIs available across the Microsoft Teams
+        "parentMessageId": "",
+        "userClickTime": 1681216672213,
+        "userFileOpenPreference": "inline",
+        "host": {
+          "name": "Teams",
+          "clientType": "web",
+          "sessionId": "a384245b-c4ee-0c76-377b-7394bd040532",
+          "ringId": "ring3_6"
+        },
+        "appLaunchId": "f7e97fbd-0b2e-477f-9599-91a5155418d5"
+      },
+      "page": {
+        "id": "Test",
+        "frameContext": "content",
+        "subPageId": "",
+        "isFullScreen": false,
+        "isMultiWindow": false,
+        "sourceOrigin": null
+      },
+      "user": {
+        "id": "4ebcc4d0-291b-4154-a85f-a89cd77aefa8",
+        "licenseType": "Unknown",
+        "loginHint": "admin@M365x654992.onmicrosoft.com",
+        "userPrincipalName": "admin@M365x654992.onmicrosoft.com",
+        "tenant": {
+          "id": "36a708ef-700d-4d60-9de0-0a5f7b7693df",
+          "teamsSku": "unknown"
+        }
+      },
+      "chat": {
+        "id": "19:meeting_OWRiMjg0N2YtNDc1Ni00YWEyLWE4YjgtODkwZTliMzczYzg1@thread.v2"
+      },
+      "meeting": {
+        "id": "MCMxOTptZWV0aW5nX09XUmlNamcwTjJZdE5EYzFOaTAwWVdFeUxXRTRZamd0T0Rrd1pUbGlNemN6WXpnMUB0aHJlYWQudjIjMA=="
+      },
+      "sharePointSite": {
+        "teamSiteUrl": "",
+        "teamSiteDomain": "m365x654992.sharepoint.com",
+        "teamSitePath": "",
+        "mySitePath": "/personal/admin_m365x654992_onmicrosoft_com",
+        "mySiteDomain": "m365x654992-my.sharepoint.com"
+      }
+    }
+   ```
+
+   ---
+
+* User type
+
+   # [Guest user](#tab/guest-user)
+
+   The following is a JSON payload response in a scheduled private meeting for a guest user :
+
+   ```json
+     {
+      "app": {
+       "locale": "en-us",
+       "sessionId": "268beeb4-a52d-4ba8-b1c8-8b9f0b9b3492",
+       "theme": "default",
+       "iconPositionVertical": 23,
+       "osLocaleInfo": {
+        "platform": "windows",
+        "regionalFormat": "en-in",
+        "longDate": "dd MMMM yyyy",
+        "shortDate": "dd-MM-yyyy",
+        "longTime": "HH:mm:ss",
+        "shortTime": "HH:mm"
+       },
+       "parentMessageId": "",
+       "userClickTime": 1678023265131,
+       "userFileOpenPreference": "inline",
+       "host": {
+        "name": "Teams",
+        "clientType": "desktop",
+        "sessionId": "967c980b-1e41-a2cd-eac0-a4bff8f73ce7",
+        "ringId": "ring1"
+       },
+       "appLaunchId": "c35c4496-f28c-4107-8e6c-2dba09fb881a"
+      },
+      "page": {
+       "id": "Test",
+       "frameContext": "content",
+       "subPageId": "",
+       "isFullScreen": false,
+       "isMultiWindow": false,
+       "sourceOrigin": NULL
+      },
+      "user": {
+       "id": "57efa5f3-273c-47e2-a871-4879e5d849cf",
+       "displayName": undefined,
+       "isCallingAllowed": undefined,
+       "isPSTNCallingAllowed": undefined,
+       "licenseType": "Unknown",
+       "loginHint": "v-prkamble@microsoft.com",
+       "userPrincipalName": "v-prkamble@microsoft.com",
```

Comment:
This is a scenario where using the sample tenants would make things clearer. I am assuming that you gathered these values from the Microsoft tenant. In that case shouldn't the Guest user be in a different tenant? 

If we picked Contoso as the host tenant so those values are used for the sharepoint details below, but Northwind is used for the guest, it would be clear what the differences are.